dist/tugboat-entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sv_stop() {
+    for s in /etc/service/*; do
+        "$RUNITDIR"/sv stop "$s"
+    done
+}
```

Comment:
This is thorough and maybe future-proofed but we do know that there is only one directory in /etc/service in the official solr image (for now), so maybe no need to loop here.